### DISCUSSION TITLE
Updating N64 firmware

### DISCUSSION THREAD

**Discussion by NUKEM0182** [darthcloud/BlueRetro#865] [Q&A] [answered]
Is there any easier methods? seems to not want to work on the OTA. 

This is the log..

Requesting Bluetooth Device...
Connecting to GATT Server...
Getting BlueRetro Service...
app_ver_is_hw2: false app_name_is_hw2: false
Init Cfg DOM...
> Bluetooth Device disconnected

I am hoping updating firmware would fix joystick sensitivity issues with xbox one controllers? its wayyy too sensitive compared to original N64 controllers. Really dont know what else to do on updating. I just have a windows 10 PC with Edge.

**Comment by NUKEM0182** [comment]
well sh*t... i think i bricked it by going through the manual way of updating with node.. all its doing is just giving the solid red and blue light and player 1 light is lit up too. tried to do steps you provided to help another guy.. holding button for  15 sec then power off after a minute. doesnt do anything. is there a discord channel for direct help? this is ridculous.

**Reply by darthcloud** [reply, marked as answer]
You mention node so I'll assume you got a retroscaler device with USB port.

Follow the instructions from the wiki to reflash:
https://github.com/darthcloud/BlueRetro/wiki/Flashing-firmware-Windows-10

**Reply by NUKEM0182** [reply]
i just now did that and brought it back to flashing state.. but i only followed it for the hw1, is HW2 the same values?

**Reply by darthcloud** [reply]
If its an external device its HW1 as noted were you downloaded the firmware.

**Reply by NUKEM0182** [reply]
The player 1-4 LED indicator isnt working right, still only shows player 1 even though 2 are connected

**Reply by darthcloud** [reply]
retroscaler didn't follow my spec and made custom FW, so led wont work with official firmware.

**Reply by NUKEM0182** [reply]
Oh, it was working fine til i updated. make sense. is there anything you can do about the joystick sensitivity?? like match the original? lol
